### PR TITLE
Adding event for missing task

### PR DESCRIPTION
**Pull request by rhuttman**

### CODE REVIEW

**Review by irsal — APPROVED** [review]

File: pallets/automation-time/src/tests.rs [hunk 1]
```diff
@@ -209,16 +229,35 @@ fn cancel_must_be_owner() {
...
-		assert_ok!(AutomationTime::cancel_task(Origin::signed(ALICE), task_id,));
 		assert_noop!(
 			AutomationTime::cancel_task(Origin::signed(ALICE), task_id),
 			Error::<Test>::TaskDoesNotExist,
 		);
 	})
 }
 
+#[test]
+fn cancel_task_not_found() {
```
Comment (irsal) [inline pallets/automation-time/src/tests.rs:243]: I don't get what this does.

Comment (rhuttman) [reply pallets/automation-time/src/tests.rs:243]: there is a weird edge case were somehow the task exists but isn't scheduled or in the overflow. There is no way this state should ever happen but if it does I want to know about it

**Review by irsal — APPROVED** [review]
quick que
